### PR TITLE
Switch to `wasm-builder` 1.0.7

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

File: test-parachains/adder/build.rs [hunk 1]
```diff
@@ -16,10 +16,16 @@
...
 fn main() {
 	build_current_project_with_rustflags(
 		"wasm_binary.rs",
-		WasmBuilderSource::Crates("1.0.5"),
+		WasmBuilderSource::Crates("1.0.7"),
 		"-C link-arg=--import-memory",
 	);
+
+	if env::var("TARGET") == Ok("wasm32-unknown-unknown".into()) {
+		println!("cargo:rustc-cfg=no_std");
```
Comment (thiolliere) [inline test-parachains/adder/build.rs]: I'm not sure I understand why this is needed. didn't you remove the no_std feature ?

**Comment by thiolliere** [comment]
also you need to remove the core feature of adder in parachain/Cargo.toml

File: test-parachains/adder/build.rs (hunk 1 above)
Comment (bkchr) [reply test-parachains/adder/build.rs]: Ahhh :( Some leftovers.

**Comment by bkchr** [comment]
> also you need to remove the core feature of adder in parachain/Cargo.toml

Which core feature?

**Comment by thiolliere** [comment]
> > also you need to remove the core feature of adder in parachain/Cargo.toml
> 
> Which core feature?

None my bad it was on my local branch

**Comment by thiolliere** [comment]
(should we backport stuff ? I don't how is strategy in this regard now)

**Pull request merged**